### PR TITLE
Adding holdings and Receiving History API tests

### PR DESCRIPTION
**Pull request by KVupp**
PURPOSE
Adding API tests for Receiving History and holdings records

APPROACH
Added new test for receiving history
Added validation method for checking if a holding record was created
method to delete the created holdings record after all the tests complete

### CODE REVIEW

**Comment by javalkar-varun** [comment]
Some of the tests are failing. Are you seeing the same failures?
screen shot 2019-02-11 at 4 57 41 pm

File: mod-orders/mod-orders.postman_collection.json [hunk 1]
```diff
@@ -7069,6 +7152,28 @@
 					"    };",
 					"",
 					"    /**",
+					"     * Validates that Holdings Record was created  in the inventory",
+					"     */",
+					"    utils.validateHoldingsRecord = function(poLine) {",
+					"        pm.expect(poLine.instance_id).to.exist;",
+					"        pm.sendRequest({",
+					"                url: utils.buildOkapiUrl(\"/holdings-storage/holdings?query=instanceId==\" + poLine.id),",
```
Comment (piotr-kalashuk) [inline mod-orders/mod-orders.postman_collection.json]: Looks like `poLine.id` should be replaced by `poLine.instance_id`

File: mod-orders/mod-orders.postman_collection.json [hunk 2]
```diff
@@ -7284,6 +7389,17 @@
...
 					"        });  ",
 					"    };",
+					"    ",
+					"    /**",
+					"     * Deletes Holdings linked by PO lines from Inventory",
+					"     */",
+					"    utils.deleteHoldingRecords = function(order) {",
+					"      order.compositePoLines.forEach(function(line) {",
+					"        if (line.instance_id) {",
+					"          utils.processDeleteRequest(\"/holdings-storage/holdings?query=instanceId=\" + line.instance_id, \"Holding should be successfully deleted\");",
```
Comment (piotr-kalashuk) [inline mod-orders/mod-orders.postman_collection.json]: According to [Holdings Storage API documentation](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/holdings-storage.html#holdings_storage_holdings_delete) only `lang` parameter is supported so the `query` is ignored. Also as far as I understand when `DELETE /holdings-storage/holdings` endpoint is called, all the holdings are being deleted.

File: mod-orders/mod-orders.postman_collection.json [hunk 3]
```diff
@@ -6562,6 +6642,7 @@
 									"    },",
 									"    function (err, res) {",
 									"        let order  = res.json();",
+									"        utils.deleteHoldingsRecord(order);",
```
Comment (piotr-kalashuk) [inline mod-orders/mod-orders.postman_collection.json]: Actually the function name is `utils.deleteHoldingRecords`

File: mod-orders/mod-orders.postman_collection.json [hunk 4]
```diff
@@ -6640,6 +6721,7 @@
...
```
Comment (piotr-kalashuk) [inline mod-orders/mod-orders.postman_collection.json]: Function is is defined as `utils.deleteHoldingRecords`

**Comment by KVupp** [comment]
@piotr-kalashuk @javalkar-varun The PR was still WIP, so I did not add reviewers too.  I created to checked if there were any merge conflicts. I will put a WIP label from next time.

**Pull request closed without merging**